__tests__/reducers/literal.test.js

```diff
@@ -1,88 +1,136 @@
+        uri:'http://schema.org/name',
+        items: [ { id: 0, content: 'Run the tests'} ]
+      }
+    })
+    expect(literalSetItems).toEqual({
+      "resourceTemplate:Monograph:Instance": {
+        'http://schema.org/name': {
+          items: [{ id: 0, content: 'Run the tests'}]
+        }
+      }
     })
 
     expect(
-      literal({
-        "formData": [{
-          "id": 1, "uri": "Run the tests", "items": []
-        }]}, {
+      setMyItems({ "resourceTemplate:Monograph:Instance": {
+        'http://schema.org/name': {
+          items: [{ id: 1, content: "Run the tests" }] },
+        'http://schema.org/description': {
+          items: []}
+        }
+      },
+      {
         type: 'SET_ITEMS',
-        payload: {id:2, uri: "add this!", items: []}
+        payload: {
+          rtId: "resourceTemplate:Monograph:Instance",
+          uri: 'http://schema.org/description',
+          items: [ { id: 2, content: "add this!"}]
+        }
       })
     ).toEqual({
-      "formData": [
-        {"id": 1, "uri": "Run the tests", "items": []},
-        {"id": 2, "uri": "add this!", "items": []}
-    ]})
+      "resourceTemplate:Monograph:Instance": {
+        'http://schema.org/name': {
+          items: [{ id: 1, content: 'Run the tests'}]
+        },
+        'http://schema.org/description': {
+          items: [{ id: 2, content: "add this!"}]
+        }
+      }
+    })
   })
   it('should handle REMOVE_ITEM', () => {
-    expect(
-      literal({formData: [{id: 1, uri:"Test", items:[
-        {content: "test content", id: 0},
-        {content: "more content", id: 1}
-        ]}]}, {
-        type: 'REMOVE_ITEM',
-        payload: {id: 0, label: "Test"}
-      })
-    ).toEqual({
-      "formData": [{
-        id: 1, uri: "Test", "items": [{content: "more content", id: 1}]
-      }]
+    expect(removeMyItem({
+      "resourceTemplate:Monograph:Instance": {
+       'http://schema.org/name': {
+         items: [
+           {content: "test content", id: 0},
+           {content: "more content", id: 1}
+         ]
+       }
+      }
+    },
+    {
+      type: 'REMOVE_ITEM',
+      payload: {
+        id: 0,
+        rtId: "resourceTemplate:Monograph:Instance",
+        uri: "http://schema.org/name",
+        content: "test content"
+      }
+    })).toEqual({
+      "resourceTemplate:Monograph:Instance": {
+        'http://schema.org/name': {
+          items: [{ id: 1, content: "more content" }]
+        }
+      }
     })
 
-
-    expect(
-      literal({formData: [
-        {id: 1, uri:"Test", items:[{content: "test content", id: 0}]},
-        {id: 2, uri:"Statement", items:[{content: "more test content", id: 0}]}
-      ]}, {
-        type: 'REMOVE_ITEM',
-        payload: {id: 0, label: "Statement"}
-      })
-    ).toEqual({
-      "formData": [
-        {"id": 1, "uri": "Test", "items": [{content: "test content", id: 0}]},
-        {"id": 2, "uri": "Statement", "items": []}
-      ]
+    expect(removeMyItem({
```

Comment:
this seems like a test showing that removing an item that doesn't match one we have already results in a no-op.  perhaps it warrants its own test description?

Reply:
Changed description per comment.